democracylab/settings.py

```diff
@@ -99,6 +99,14 @@
 
 LOGIN_REDIRECT_URL = '/'
 
+#Caching number of tag counts only for now - change this if other things are db-cached
```

Comment:
Copy/pasted the wrong thing into my commit message. It should read this: May need to run `python manage.py createcachetable` per the [documentation](https://docs.djangoproject.com/en/2.1/topics/cache/#database-caching).